mitmproxy/utils/debug.py

```diff
@@ -5,18 +5,24 @@
 import signal
 import platform
 import traceback
+import subprocess
 
 from mitmproxy import version
 
 from OpenSSL import SSL
 
 
-def sysinfo():
+def dump_system_info():
+    old_dir = os.getcwd()
+    os.chdir(os.path.abspath(os.path.join(os.path.dirname(__file__), "..")))
+    git_describe = subprocess.check_output(['git', 'describe', '--tags', '--long']).decode().strip()
```

Comment:
Let's wrap this in a catch-all try-except block. Users may not have git installed, etc. etc. If it fails, we can still display everything else.